platform/darwin/src/MGLAbstractShapeSource.mm

```diff
@@ -113,5 +115,22 @@ @implementation MGLAbstractShapeSource
         }
         sourceOptions.tileOptions.tolerance = value.doubleValue;
     }
+
+    if (NSNumber *value = options[MGLShapeSourceOptionWrapCoordinates]) {
```

Comment:
Nitpick: If it doesn't already, we should probably warn on assignments in conditionals.